### DISCUSSION TITLE
I wish the Player Detector unable to detect players in spectator mode.

### DISCUSSION THREAD

**Discussion by HoracecaroH** [TechReborn/TechReborn#3209] [Ideas]
Player Detector can detect players in spectator mode. However, players in spectator mode should not interact with the world. So I want Player Detector not to detect players in spectator mode.

**Comment by HoracecaroH** [comment]
To add: My Minecraft version:1.18.2 fabric 
TechReborn version:5.2.1

**Reply by xanthian** [reply]
Not likely to be fixed for this version.
Only the most recent MC release is maintained unless there's a catastrophic issue
